protocol/communities/community.go

```diff
@@ -2312,6 +2312,31 @@ func (o *Community) ValidateEvent(event *CommunityEvent, signer *ecdsa.PublicKey
 }
 
 func (o *Community) MemberCanManageToken(member *ecdsa.PublicKey, token *community_token.CommunityToken) bool {
-	return o.IsMemberOwner(member) || (o.IsMemberTokenMaster(member) &&
+	return o.IsMemberOwner(member) || o.IsControlNode() || (o.IsMemberTokenMaster(member) &&
```

Comment:
I don't think that we need to do a check of `o.IsControlNode()` as controlNode must be by default and Owner. It if is not right - then there is a bug somewhere